ref_builder/repo.py

```diff
@@ -366,6 +366,34 @@ def replace_sequence(
 
         return new_sequence
 
+    def link_sequence(
+            self,
+            otu_id: uuid.UUID,
+            isolate_id: uuid.UUID,
+            sequence_id: uuid.UUID,
+            original_isolate_id: uuid.UUID
+    ):
+        """Link an existing sequence to the given isolate."""
+        otu = self.get_otu(otu_id)
+
+        original_isolate = otu.get_isolate(original_isolate_id)
+        if original_isolate is None:
+            raise ValueError("Original isolate not found.")
```

Comment:
If this isolate is ever deleted (we intend to support this), the reference will be broken.

Options:

1. Duplicate the sequence, including ID in both the event stream and snapshot. When a sequence is updated, write the change in both locations. When it is deleted, only do so in the location it 
2. Decouple sequences from OTUs.

We can discuss tomorrow.